### ISSUE TITLE
Mic "access" while running in background on android 14

### ISSUE THREAD

**Issue by Bbugreporter** [iNPUTmice/Conversations#4464]
The app seems to lose access to the mic on android 14, once the app is running in background.

**Comment by licaon-kter** [comment]
In the future report issues on Codeberg as the READMEsays.

But, can you rephrase, isn't it expected to be able to access the mic during a call?

**Comment by Bbugreporter** [comment]
Got it codeberg. I made a few tests yesturday, here is what I found. During audio calls, locking the screen or putting the conversations app in the backround cuts the mic, no sound on the other end. The "mic in use" icon in the status bar simply disappears. During video calls this happens only when the screen gets locked but the video keep working as intended.
The bug is reproducible on android 13 as well.
I hope that helps.

**Comment by licaon-kter** [comment]
No logs, no bug. :)

Did you except Conversations from battery optimisations ( Android Settings - Apps - Conversations - Battery - No Restrictions)?

Did you enable Mic access to "Always when using app"?

Which device? Which Android flavor (stock, custom)?
